api/DedicatedWorkerGlobalScope.json

```
@@ -52,31 +52,31 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/close",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "4"
             },
             "chrome_android": {
               "version_added": true
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "12"
             },
             "firefox": {
-              "version_added": "54"
+              "version_added": "3.5"
             },
             "firefox_android": {
-              "version_added": "54"
+              "version_added": "4"
             },
             "ie": {
-              "version_added": null
+              "version_added": "10"
             },
             "opera": {
-              "version_added": true
+              "version_added": "11.5"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": null
```

Given, Opera's history, I think it's safe to mirror from desktop:

```suggestion
              "version_added": "11.5"
```